src/crimson/mon/MonClient.h

```diff
@@ -44,10 +44,7 @@ struct MMonCommandAck;
 struct MLogAck;
 struct MConfig;
 
-enum class log_flushing_t {
-  NO_FLUSH,
-  FLUSH
-};
+enum class log_flushing_t;
```

Comment:
why leaving it here?

Reply:
forward declaration for line 73:
`  seastar::future<> send_log(log_flushing_t flush_flag);`